.changeset/pink-berries-try.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/test-utils": major
+---
+
+MockLogger is no longer a TelemetryLogger
```

Comment:
For this package the referenced logger should be `EventAndErrorTrackingLogger` instead of `MockLogger`, and part of the note below should move to the changeset for telemetry-utils.

Reply:
fixed it, as it is intended for @fluidframework/telemetry-utils

Reply:
Then do we need another changeset for test-utils and `EventAndErrorTrackingLogger`? It also stops implementing `TelemetryLogger` and the package is technically public.

Reply:
I think it is very unlikely it is used, but i can add one.